### PR TITLE
AAP-12257: Updated RN to include v2.4 content only

### PR DESCRIPTION
**Pull request by sayjadha**
This PR is a continuation of AAP v2.4 Release Notes restructuring efforts, and the JIRA is https://issues.redhat.com/browse/AAP-12257. 
This PR also addresses #1 of JIRA https://issues.redhat.com/browse/AAP-14684.

Changes made:

- Included AAP v.24 specific info. only. Removed info. about all older versions. They were restructured in other PRs. 
- Moved 'Platform Resource Operator' enhancement to Tech. Preview section per a customer req. See JIRA https://issues.redhat.com/browse/AAP-14684.

### CODE REVIEW

File: downstream/titles/release-notes/topics/aap-24.adoc [hunk 1]
```diff
@@ -101,6 +101,17 @@ Following are the technology preview features:
 
 * Ansible Automation Platform 2.4 adds the ability to install the automation controller for IBM Power (ppc64le), IBM Z (s390x), and IBM(R) LinuxONE (s390x) architectures.
 
+* Starting with Ansible Automation Platform 2.4, the Platform Resource Operator can be used to create the following resources in automation controller by applying YAML to your Openshift cluster:
```
Comment (dcdacosta) [inline downstream/titles/release-notes/topics/aap-24.adoc:104]: Openshift should be OpenShift.

**Review by dcdacosta — APPROVED** [review]
One minor change; otherwise, LGTM.

File: downstream/titles/release-notes/topics/aap-24.adoc (hunk 1 above)
Comment (sayjadha) [reply downstream/titles/release-notes/topics/aap-24.adoc]: Made the edit, thank you!

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**